crates/rolldown_binding/src/options/plugin/binding_builtin_plugin.rs

```diff
@@ -88,14 +88,35 @@ pub struct BindingModulePreloadPolyfillPluginConfig {
 }
 
 #[napi_derive::napi(object)]
-#[derive(Debug, Deserialize, Default)]
-#[serde(rename_all = "camelCase")]
+#[derive(Debug, Default)]
```

Comment:
When is `Deserialize` used? Is it fine to simply remove it?

Reply:
Yeah. It could be removed. It's basically for `#[serde(rename_all = "camelCase")]`. Seems napi-rs supports this by default.